### ISSUE TITLE
Feature request: Integrate with Entity Filter 

### ISSUE THREAD

**Issue by alexisml** [jcwillox/lovelace-paper-buttons-row#12]
It would be nice being able to show/hide buttons the same way entity filter enables you to do so.

https://www.home-assistant.io/lovelace/entity-filter/

**Comment by alexisml** [comment]
I'm referring to the second yes :) 
Oh I see what you mean, with styles yes. I'll give it a try and come back.

Thanks!

**Comment by alexisml** [comment]
Ok I made it work with styles. For some reason styles are shared across all items related to the same entity... Is that expected?

`  - buttons:
      - entity: input_boolean.tv_living_control_full
        icon: 'mdi:remote-tv'
        name: false
        state_styles:
          'on':
            icon:
              color: var(--paper-item-icon-active-color)
    type: 'custom:paper-buttons-row'
  - buttons:
      - entity: input_boolean.tv_living_control_full
        icon: 'mdi:numeric-1-circle-outline'
        name: false
        state_styles:
          'off':
            button:
              display: none
      - entity: input_boolean.tv_living_control_full
        icon: 'mdi:numeric-2-circle-outline'
        name: false
        state_styles:
          'off':
            button:
              display: none
      - entity: input_boolean.tv_living_control_full
        icon: 'mdi:numeric-3-circle-outline'
        name: false
        state_styles:
          'off':
            button:
              display: none`

(Bottom 3 buttons are shown yellow).

**Comment by jcwillox** [comment]
The styles should not be shared, in your case it's just that `var(--paper-item-icon-active-color)` is the default colour for active buttons, you can check this by changing the first config to `color: var(--primary-color)` and you'll notice that the top button will be blue (depending on theme) and the others will still be yellow

**Comment by alexisml** [comment]
You are absolutely right sir. Thanks.

**Comment by ktownsend-personal** [comment]
I have a situation where I need to hide a button based on state of a different entity. I can't use `state_styles` and `display: none` in this case. Any chance we could add config to hide a button based on a different entity state? I'm using a variation of the `conditions` feature of the `picture elements` card in my example, but it can be done differently. I chose to do it both on the overall element and on the button because sometimes the whole group should be hidden, and other times just one part of a group should be hidden. The `placeholder` in my example is for different situations where I may want the gap to remain, or I may want the adjacent elements to slide over in its place. 

```yaml
# example

- type: custom:paper-buttons-row
  conditions:
    - entity: switch.filter_pump
      state: 'on'
  buttons:
    - entity: switch.spa_mode # always available if filter pump is running
    - entity: switch.spa_blower # only available if spa_mode is also turned on and something else is turned off
      placeholder: true
      conditions:
        - entity: switch.spa_mode
          state: 'on'
        - entity: switch.something_else
          state_not: 'off' # this is nice for reversing condition
```

**Comment by alexisml** [comment]
@ktownsend-personal you can use `entity` for the `state_styles` and then use a different `entity` in the `tap_action`. 

```
# example

- type: custom:paper-buttons-row
  conditions:
    - entity: switch.filter_pump
      state: 'on'
  buttons:
    - entity: sensor.filter_pump # whatever is called
      state_styles:
          'off':
            button:
              display: none
        tap_action:
          action: toggle
          entity: switch.spa_mode
```

**Comment by ktownsend-personal** [comment]
@alexisml interesting solution. I could override the name and icon to make it look like the target device, but I wouldn't get the right icon state of the device being controlled. I think conditions would work much better.

**Comment by jcwillox** [comment]
This should be solved by release [0.5.0](https://github.com/jcwillox/lovelace-paper-buttons-row/releases/tag/0.5.0).
